<launch>

  <node pkg="robot_state_publisher" exec="robot_state_publisher">
    <param name="robot_description" value="$(command '$(exec-in-pkg xacro xacro) $(find-pkg-share turtle_brick)/turtle.urdf.xacro')" /> -->
  </node>

  <node pkg="turtle_brick" exec="run_turtle" name="run_turtle">
    <remap from="cmd_vel" to="turtle1/cmd_vel" />
  </node>

  <!-- rviz will enable us to see the robot -->
  <node pkg="rviz2" exec="rviz2" args="-d $(find-pkg-share turtle_brick)/turtle_arena.rviz"/>
</launch>
